packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.tsx

```diff
@@ -62,13 +64,15 @@ export class ChoiceGroup extends BaseComponent<IChoiceGroupProps, IChoiceGroupSt
     let { label, options, className, required } = this.props;
     let { keyChecked, keyFocused } = this.state;
 
+    let nativeProps = getNativeProps(this.props, inputProperties);
```

Comment:
is there a reason you're using this here? If you're not mixing the props in, you don't need the filter function.

All I see being done with nativeProps is accessing `[aria-labelledby']` which can be done on `this.props`.